src/coreclr/jit/hwintrinsiccodegenxarch.cpp

```diff
@@ -758,6 +758,9 @@ void CodeGen::genHWIntrinsic(GenTreeHWIntrinsic* node)
         case InstructionSet_AVX512BW_VL:
         case InstructionSet_AVX512VBMI:
         case InstructionSet_AVX512VBMI_VL:
+        case InstructionSet_AVX10v1:
+        case InstructionSet_AVX10v1_V256:
```

Comment:
Is this one relevant still or can it be removed?

Reply:
These are needed afaik. I saw some compilation issues and test failures when these cases were absent. But they were mainly present for ```LeadingZeroCount``` and ```DetectCOnflicts```. I will retest and post a comment later.